undead-discord.css

```diff
@@ -79,7 +79,7 @@
 /* .status needs to be before .status-typing */
 
 .status {
-  border-radius: 33% !important;
+  border-radius: 0px !important;
```

Comment:
Because of the global `* { border-radius: unset !important; }` it is not required to set the border radius to 0, but you can just remove the block.

The following block that affects `.status-typing` should also be removed.